styles.css

```diff
@@ -138,7 +141,7 @@ button {
   flex-direction: column;
   width: 30%;
   height: 325px;
-  margin: 1%;
```

Comment:
Great job trouble shooting this and doing the math to find the exact percentage.